src/meshapi/util/events/osticket_creation.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import time
+
+import requests
+from django.db.models.base import ModelBase
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from meshapi.models import Install
+
+OSTICKET_API_TOKEN = os.environ.get("OSTICKET_API_TOKEN")
+OSTICKET_NEW_TICKET_ENDPOINT = os.environ.get("OSTICKET_NEW_TICKET_ENDPOINT")
```

Comment:
```suggestion
OS_TICKET_URL = os.environ.get("OSTICKET_URL", "https://support.nycmesh.net")
OSTICKET_NEW_TICKET_ENDPOINT = f"{OS_TICKET_URL}/api/http.php/tickets.json"
```

Reply:
Hmm the one drawback to this approach is that we won't be able to link to prod tickets from dev MeshDB, without _also_ cutting new tickets from installs created in dev to prod (super bad, the synthetics tests make new installs every 5 minutes)

Reply:
I see, good call